### PR TITLE
executor: migrate test-infra to testify for show_test.go

### LINKED ISSUE #28612: migrate test-infra to testify for `show_test.go`

### PR DESCRIPTION
**Pull request by hawkingrei**
Signed-off-by: Weizhen Wang <wangweizhen@pingcap.com>

### What problem does this PR solve?

Issue Number: close #28612

Problem Summary:

### Check List

Tests 

- [ ] Unit test
- [ ] Integration test
- [ ] Manual test (add detailed scripts or steps below)
- [ ] No code

Side effects

- [ ] Performance regression: Consumes more CPU
- [ ] Performance regression: Consumes more Memory
- [ ] Breaking backward compatibility

Documentation

- [ ] Affects user behaviors
- [ ] Contains syntax changes
- [ ] Contains variable changes
- [ ] Contains experimental features
- [ ] Changes MySQL compatibility

### Release note

```release-note
None
```

### CODE REVIEW

**Review by tisonkun — APPROVED** [review]

File: executor/show_test.go [hunk 1]
```diff
@@ -71,14 +68,16 @@ func (s *testSuite5) TestShowVisibility(c *C) {
 	// Grant any global privilege would make show databases available.
 	tk.MustExec(`grant CREATE on *.* to 'show'@'%'`)
 	rows := tk1.MustQuery("show databases").Rows()
-	c.Assert(len(rows), GreaterEqual, 2) // At least INFORMATION_SCHEMA and showdatabase
+	require.GreaterOrEqual(t, len(rows), 2)
```
Comment (disksing) [inline executor/show_test.go:71]: do we need to keep the comment?

Comment (hawkingrei) [reply executor/show_test.go:71]: I think we can remove it.

**Comment by hawkingrei** [comment]
@disksing PTAL

**Review by disksing — APPROVED** [review]

**Review by tangenta — APPROVED** [review]

**Pull request merged**